rules/instance_network_interface_rules.yaml

```diff
@@ -0,0 +1,13 @@
+rules:
```

Comment:
Also, make sure this can be applied just at a folder level and it's projects, rather than only at an org level or per project level.